remme/rpc_api/transaction.py

```diff
@@ -203,6 +205,29 @@ def _get_proto_validator(current_handler, tr_payload_pb):
     head = request.params.get('head')
     reverse = request.params.get('reverse')
 
+    if ids is not None:
+        for identifier in ids:
+            try:
+                return await client.fetch_batch(identifier)
```

Comment:
Are there tests for each added `return await client.fetch_batch(identifier)` logic?

Reply:
Done.

Reply:
@Art17, use method `list_batches`, please.

Reply:
Done.